tests/test_db_setup.py

```diff
@@ -213,6 +213,49 @@ def test_a():
         result.stdout.fnmatch_lines(["*PASSED*test_a*"])
 
 
+class TestSqliteWithMultipleDbsAndXdist:
+
+    db_settings = {
+        "default": {
+            "ENGINE": "django.db.backends.sqlite3",
+            "NAME": "/tmp/should-not-be-used",
+        },
+        "db2": {
+            "ENGINE": "django.db.backends.sqlite3",
+            "NAME": "db_name",
+            "TEST": {"NAME": "test_custom_db_name"},
+        }
+    }
+
+    def test_sqlite_database_renamed(self, django_testdir):
+
+        django_testdir.create_test_module(
+            """
+            import pytest
+            from django.db import connections
+
+            @pytest.mark.django_db
+            def test_a():
+                (conn1, conn2) = connections.all()
+
+                if conn1.alias == 'db2':
+                    conn1, conn2 = conn2, conn1
```

Comment:
Is this switching done for predictable order?

Reply:
Yes, it's just for that. I can sort `connections.all()` instead to simplify the code.

Reply:
Would be good, yes.

Reply:
Or add a comment.